services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],

```

Comment:
The type annotation `list[dict]` is used, which is only supported in Python 3.9 and later. If you need to support earlier versions, consider using `List[Dict]` from the `typing` module.